tao/tao_bridge/passes/tao_optimization_pass.cc

```diff
@@ -113,20 +113,6 @@ Status TaoOptimizationPass::Run(const GraphOptimizationPassOptions& options) {
 
   bool mlir_whole_graph_compilation =
       GetTaoBridgeOptions()->experimental_enable_mlir_whole_graph_compilation;
-  auto flags = GetMarkForCompilationPassFlags();
-  if (flags->tf_xla_cpu_global_jit) {
```

Comment:
not delete this part, we still need this on cpu.

Reply:
Ok,  but `TaoCloneConstantsForBetterClusteringPass` will be deleted.

Reply:
TaoCloneConstantsForBetterClusteringPass reserved.